packages/components/src/inputs/src/NumberField.tsx

```diff
@@ -166,7 +166,7 @@ function NumberField(props: NumberFieldProps, ref: ForwardedRef<HTMLDivElement>)
     const inputMarkup = (
         <ClearContainerSlots>
             <InputGroup
-                isFluid={isFluid}
+                isFluid
```

Comment:
NumberField are fluid by default now?

Reply:
this was a change I made everywhere else but forgot here. It's the Input group that it fluid, not the field. Basically, the width is determined by the field and we want the input group to expand to this width.

Reply:
Perfect